cloudant-client/src/main/java/com/cloudant/client/org/lightcouch/CouchDatabaseBase.java

```diff
@@ -348,6 +348,41 @@ public Response saveAttachment(InputStream in, String name, String contentType,
     }
 
     /**
+     * Removes an attachment from a document.
+     * <p>The object must have the correct <code>_id</code> and <code>_rev</code> and <code>attachmentName</code> values.
+     *
+     * @param object The document to remove attachment as object.
+     * @param attachmentName The attachment name to remove.
+     * @return {@link Response}
+     * @throws NoDocumentException If the document is not found in the database.
+     */
+    public Response removeAttachment(Object object, String attachmentName) {
+        assertNotEmpty(object, "object");
+        JsonObject jsonObject = getGson().toJsonTree(object).getAsJsonObject();
+        final String id = getAsString(jsonObject, "_id");
+        final String rev = getAsString(jsonObject, "_rev");
+        return removeAttachment(id, rev, attachmentName);
+    }
+
+    /**
+     * Removes an attachment from a document given both a document <code>_id</code> and
+     * <code>_rev</code> and <code>attachmentName</code> values.
+     *
+     * @param id  The document _id field.
+     * @param rev The document _rev field.
+     * @param attachmentName The document attachment field.
+     * @return {@link Response}
+     * @throws DocumentConflictException
```

Comment:
I think `NoDocumentException` should be in the throws list for the documentation.